src/app/static/src/tests/components/runReport/parameterList.test.ts

```diff
@@ -56,7 +56,7 @@ describe(`run report parameter list`, () => {
         const inputs = wrapper.find("table").findAll("input")
         await inputs.at(0).setValue("test Value1")
         expect(wrapper.emitted().paramsChanged.length).toBe(2)
-        expect(wrapper.emitted().paramsChanged[0]).toMatchObject([innerParams, true])
+        expect(wrapper.emitted().paramsChanged[1]).toMatchObject([innerParams, true])
```

Comment:
Sorry, I meant you need to test that the params emitted have been updated - so shouldn't match innerParams, as the first param value should have been updated to "test Value1